nlmod/read/ahn.py

```diff
@@ -303,15 +297,80 @@ def get_ahn4_tiles(extent=None):
     return gdf
 
 
+def _get_tiles_ellipsis(
+    extent=None,
+    crs=28992,
+    timeout=120.0,
+    base_url="https://api.ellipsis-drive.com/v3",
+    path_id="a9d410ad-a2f6-404c-948a-fdf6b43e77a6",
+    timestamp_id="87a21a71-c39f-4e92-a43b-207bc7dfe714",
+):
+    url = f"{base_url}/path/{path_id}/vector/timestamp/{timestamp_id}/listFeatures"
+
+    r = requests.get(url, timeout=timeout)
+    if not r.ok:
+        raise (HTTPError(f"Request not successful: {r.url}"))
+    gdf = gpd.GeoDataFrame.from_features(r.json()["result"]["features"], crs=4326)
+    gdf = gdf.to_crs(crs)
```

Comment:
The transform here is happening using EPSG 28992, not our custom definition. Are we sure things are ending up in the right spot in NL?